KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -138,6 +138,14 @@ recursive_merge(x...) = x[end]
 #Sequence object functions
 size(x::Sequence) = size(x.GR[1,:])
 
+"""
+For comparing two `Sequence`s custom types
+"""
+Base.isapprox(s1::Sequence, s2::Sequence) = begin
```

Comment:
The syntax `Base.isapprox` instead of `import Base: isapprox` is not consistent with the rest of the code. Julia Blue (#187) recommends `Base.isapprox`, we need to choose one or the other.